frontend/src/pages/modelRegistry/ModelRegistryRoutes.tsx

```diff
@@ -3,14 +3,21 @@ import { Navigate, Route } from 'react-router-dom';
 import ProjectsRoutes from '~/concepts/projects/ProjectsRoutes';
 import ModelRegistryCoreLoader from './ModelRegistryCoreLoader';
 import ModelRegistry from './screens/ModelRegistry';
+import { ModelVersionsTabs } from './screens/const';
+import ModelVersions from './screens/ModelVersions';
 
-const ModelServingRoutes: React.FC = () => (
+const ModelRegistryRoutes: React.FC = () => (
   <ProjectsRoutes>
     <Route path={'/:modelRegistry?/*'} element={<ModelRegistryCoreLoader />}>
       <Route index element={<ModelRegistry />} />
+      <Route path="modelregistry-sample" element={<ModelRegistry />} />
+      <Route
+        path="modelregistry-sample/registered_models/:registeredModelId"
+        element={<ModelVersions tab={ModelVersionsTabs.VERSIONS} empty={false} />}
+      />
```

Comment:
@manaswinidas, why aren't we taking the registry name as a parameter instead of defining it here as a constant?

Reply:
this was the first change I made after an internal conversation. Included it now.